linkerd/app/src/lib.rs

```diff
@@ -142,6 +143,15 @@ impl Config {
             info_span!("dst").in_scope(|| dst.build(dns, metrics, identity.receiver().new_client()))
         }?;
 
+        let profiles = svc::stack(profiles::GetProfile::into_service(dst.profiles))
+            // TODO(eliza): these configs should not be inbound/outbound-specific,
+            // and should also have names that remotely reflect what they configure...
```

Comment:
i thought it was better to make the environment variable change in a separate PR, since it would presumably also require a proxy-injector change. but i could be convinced to do it here, if we'd prefer not to let this TODO make it to `main`?